components/forms/change-password/index.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useRef } from 'react'
+
+import styles from './styles.css'
+
+import { Card } from 'design'
+import { withGlobalStore } from 'store'
+
+const { IDP_URL } = process.env
+
+const ChangePassword = ({ className, store, tag = 'section' }) => {
+  const iframeRef = useRef(null)
+  const searchParams = {
+    identity_provider_url: IDP_URL,
+    email: store.user?.email,
+  }
+
+  // remove empty params
+  Object.keys(searchParams).forEach(
+    (key) => searchParams[key] === undefined && delete searchParams[key]
+  )
```

Comment:
Try to avoid deleting props. It's not clear and does not perform very well.

```suggestion
  const email = store.user?.email
  const searchParams = {
    ...(email != null ? { email } : {}),
    identity_provider_url: IDP_URL,
  }
```